packages/esm-patient-common-lib/src/dashboards/DashboardExtension.tsx

```diff
@@ -7,12 +7,11 @@ import styles from './dashboardextension.scss';
 export interface DashboardExtensionProps {
   title: string;
   basePath: string;
-  currentPath: string;
+  currentPath?: string;
```

Comment:
Hey sorry just saw this.  @ibacher 's change [here](https://github.com/openmrs/openmrs-esm-patient-chart/pull/733/files) makes this change obsolete.